server/prisma/generator/factories/user.factory.ts

```diff
@@ -19,27 +19,31 @@ const createUsers = async (
     email: 'foo@bar.com',
     name: 'The Owner',
     instance_role: { connect: { id: instanceRoles.owner.id } },
+    image_url: '',
```

Comment:
```suggestion
```

Reply:
Oh 😅, they were an attempt to please typescript, is there a better way around that type error?🤔